scanner/src/test/kotlin/ScanCodeTest.kt

```diff
@@ -139,282 +140,573 @@ class ScanCodeTest : WordSpec({
-            val resultFile = File("src/test/assets/esprima-2.7.3_scancode-2.2.1.json")
-            val result = scanner.getRawResult(resultFile)
-
-            val expectedFindingBsd2 = LicenseFindings(
-                "BSD-2-Clause",
-                sortedSetOf(
-                    TextLocation("esprima.js", 4, 22),
-                    TextLocation("LICENSE.BSD", 3, 21),
-                    TextLocation("bin/esvalidate.js", 5, 23),
-                    TextLocation("bin/esparse.js", 5, 23),
-                    TextLocation("tools/generate-fixtures.js", 3, 19),
-                    TextLocation("test/check-complexity.js", 4, 22),
-                    TextLocation("test/regression-tests.js", 4, 22),
-                    TextLocation("test/downstream.js", 4, 22),
-                    TextLocation("test/browser-tests.js", 4, 22),
-                    TextLocation("test/profile.js", 4, 22),
-                    TextLocation("test/unit-tests.js", 4, 22),
-                    TextLocation("test/check-version.js", 6, 24),
-                    TextLocation("test/grammar-tests.js", 4, 22),
-                    TextLocation("test/benchmarks.js", 4, 22),
-                    TextLocation("test/utils/evaluate-testcase.js", 4, 22),
-                    TextLocation("test/utils/create-testcases.js", 4, 22),
-                    TextLocation("test/utils/error-to-object.js", 4, 22)
-                ),
-                sortedSetOf(
-                    CopyrightFindings(
-                        "Copyright (c) jQuery Foundation, Inc. and Contributors",
-                        sortedSetOf(
-                            TextLocation("LICENSE.BSD", 1, 1),
-                            TextLocation("bin/esparse.js", 3, 3),
-                            TextLocation("bin/esvalidate.js", 3, 3),
-                            TextLocation("esprima.js", 2, 2),
-                            TextLocation("test/benchmarks.js", 2, 2),
-                            TextLocation("test/browser-tests.js", 2, 2),
-                            TextLocation("test/check-complexity.js", 2, 2),
-                            TextLocation("test/check-version.js", 4, 4),
-                            TextLocation("test/downstream.js", 2, 2),
-                            TextLocation("test/grammar-tests.js", 2, 2),
-                            TextLocation("test/profile.js", 2, 2),
-                            TextLocation("test/regression-tests.js", 2, 2),
-                            TextLocation("test/unit-tests.js", 2, 2),
-                            TextLocation("test/utils/create-testcases.js", 2, 2),
-                            TextLocation("test/utils/error-to-object.js", 2, 2),
-                            TextLocation("test/utils/evaluate-testcase.js", 2, 2),
-                            TextLocation("tools/generate-fixtures.js", 2, 2)
-                        )
+    "generateSummary()" should {
+        // TODO: minimize this test case
```

Comment:
Please use properly formatted sentences even for TODOs: "Minimize this test case."

Reply:
done.